content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
 
 # Compute Engine
 
+Ar-Compute-engine is the argo component responsible for performing various transformations and computations on the collected metric data in order to provide availability and reliability metrics. The results produced by the compute-engine are forwarded and stored to the configured datastore (MongoDB)
+
+Ar-Compute-engine uses the hadoop software stack for performing calculations on the metric data as map-reduce jobs. These jobs can also be run locally (single node mode) in the absence of a proper hadoop cluster. Under the hood the engine uses Apache Pig for map-reduce job submission and execution.
+
+The main engine's input is the metric data collected from the broker network. These files (according to the default configuration of ar-consumer component) reside to the `/var/lib/ar-consumer/` folder.
+
+This core metric data set is processed and transformed with aditional information provided by the ar-sync components. Additional information includes topology, grouping of services, weight factors, lists relevant metrics to be considered, etc. This information is (by default) provided per-tenant/per-job in the following path
+`/var/lib/ar-sync/{tenant-name}/{job-name}`
+
+for e.g. for tenant-name=T1 and job-name=JobA the correct path with the sync files is as follows
+`/var/lib/ar-sync/T1/JobA`
+
+Some sync files that concern the whole enviroment such as the downtime information are provided once in the root ar-sync folder `/var/lib/ar-sync/`
+
+In order for the engine to be able to connect and submit jobs successfully in a hadoop cluster, proper hadoop client configuration files must be present the installed node (`/etc/hadoop/conf/`)
+
+### Configuration files of ar-compute-engine
+
+With the installation of ar-compute-engine component a main configuration file is deployed in `/etc/ar-compute-engine.conf`. In addition, a directory with supplementary secondary configuration files is created in `/etc/ar-compute/`
+
+##### /etc/ar-compute-engine.conf (Main configuration file)
+
+This file includes various global parameters used by the engine which are organized in sections, as described next:
+
+
+
+#####parameters:
+######section: `[default]`
+
+- `mongo_host={IP ADDRESS}`  
+specify the ip address of the datastore node (running mongodb)
+- `mongo_port={PORT_NUMBER}`   
```

Comment:
What about the user and password for the connection to the mongodb? Also how does this work with the multitenant support?